DEBIFY_IMAGE

```diff
@@ -1 +1 @@
-registry.tld/conjurinc/debify
+registry2.itci.conjur.net/conjurinc/debify:3.0.0.2-260d751
```

Comment:
Should be switched back after debify release

Reply:
Will rollback before push to master

Reply:
Fixed